pims/bioformats.py

```diff
@@ -5,32 +5,53 @@
 from pims.base_frames import FramesSequence
 from pims.frame import Frame
 from warnings import warn
-from os.path import isfile
+import os
 
 try:
-    import javabridge
+    import jpype
 except ImportError:
-    javabridge = None
+    jpype = None
 
-try:
-    import bioformats
-except ImportError:
-    bioformats = None
+LOCI_TOOLS_PATH = os.path.join(os.path.dirname(__file__),
```

Comment:
This should probaly use one of the XDG paths.  This works for conda or develop installs as you have write permission to the install location, but if pims is installed system wide this directory could be read-only.  This should probably also be a search cascade which leads to a rather annoying type of pain, see https://github.com/NSLS-II/metadatastore/blob/master/metadatastore/conf.py